composer/algorithms/ghost_batchnorm/metadata.json

```diff
@@ -3,7 +3,7 @@
         "name": "Ghost BatchNorm",
         "class_name": "GhostBatchNorm",
         "functional": "apply_ghost_batchnorm",
-        "tldr": "Use smaller samples to compute batchnorm",
+        "tldr": "Use smaller # samples to compute batchnorm",
```

Comment:
Does this change the meaning? e.g., smaller number of samples is smaller batch vs smaller samples is smaller image size?

Reply:
nvm. Acutally you added # as opposed to removing it so it's clear now.